docs/documentation/server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

```diff
@@ -101,6 +101,23 @@ This is used by clients running on internet-connected devices that have limited
 . The application provides the user with the user code and the verification URI. The user accesses a verification URI to be authenticated by using another browser. You could define a short verification_uri that will be redirected to {project_name} verification URI (/realms/realm_name/device)outside {project_name} - fe in a proxy.
 . The application repeatedly polls {project_name} to find out if the user completed the user authorization. If user authentication is complete, the application exchanges the device code for an _identity_, _access_ and _refresh_ token.
 
+[[_assertion_grant]]
+===== JWTs as Authorization Grants
+
+The JWTs as authorization grants flow described in https://www.rfc-editor.org/rfc/rfc7523#section-2.1[RFC 7523 Section 2.1] enables trusted entities to authenticate users by submitting a signed JWT to the OIDC token endpoint.
+
+The protocol works as follows:
+
+. A trusted entity generates a JWT, specifying the username of the user in the _sub_ claim of the JWT
```

Comment:
The username from the assertion does not necessarily map directly to a username in Keycloak.